File: SeatGeekSearch/app/src/main/java/com/cjwilliams24680/seatgeeksearch/models/EventStats.java
package com.cjwilliams24680.seatgeeksearch.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by chris on 4/11/18.
 */

public class EventStats {

    @Expose
    @SerializedName("listing_count")
    int listingCount;

    @Expose
    @SerializedName("lowest_price")
    int lowestPrice;

    public EventStats() { }

    public int getListingCount() {
        return listingCount;
    }

    public int getLowestPrice() {
        return lowestPrice;
    }

}

File: SeatGeekSearch/app/src/main/java/com/cjwilliams24680/seatgeeksearch/network/models/EventStats.kt
package com.cjwilliams24680.seatgeeksearch.network.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by chris on 4/11/18.
 */
@Parcelize
data class EventStats(
        @SerializedName("listing_count")
        val listingCount: Int,
        @SerializedName("lowest_price")
        val lowestPrice: Int
) : Parcelable
